<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>read_xy</class>
 <widget class="QWidget" name="read_xy">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>300</y>
     <width>160</width>
     <height>165</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="x_min_label">
        <property name="text">
         <string>X Min?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_min"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="x_max_label">
        <property name="text">
         <string>X Max?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="x_max"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="y_min_label">
        <property name="text">
         <string>Y Min?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y_min"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="y_max_label">
        <property name="text">
         <string>Y Max?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="y_max"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="confirm">
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="points_table">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>50</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="custom_graphics" name="image_view">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>Delete Current Row</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clear">
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="export_button">
      <property name="text">
       <string>Export CSV</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>custom_graphics</class>
   <extends>QGraphicsView</extends>
   <header>custom_graphics.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
